tg_bot/modules/weather.py

```diff
@@ -0,0 +1,74 @@
+import pyowm
+from pyowm import timeutils, exceptions
+
+from typing import Optional, List
+from tg_bot import dispatcher, updater, API_WEATHER
+from tg_bot.modules.disable import DisableAbleCommandHandler
+
+from telegram import Message, Chat, Update, Bot
+from telegram.ext import run_async
+
+@run_async
+def cuaca(bot: Bot, update: Update, args: List[str]):
+    location = " ".join(args)
+    if location.lower() == bot.first_name.lower():
+        update.effective_message.reply_text("I will keep an eye on both happy and sad times!")
+        bot.send_sticker(update.effective_chat.id, BAN_STICKER)
+        return
+
+    try:
+        bot.sendChatAction(update.effective_chat.id, "typing") # Bot typing before send message
```

Comment:
I'm against sending chat actions, as it's unnecessary and causes extra api calls; it just slows down the bot.